### DISCUSSION TITLE
N1 编译时总是出现这个错误

### DISCUSSION THREAD

**Discussion by brianzc0306** [coolsnowwolf/lede#12251] [Bugs / 问题反馈]
帮忙看一下怎样解决
error: failed to parse bitcode for LTO module: Invalid alignment value (Producer: 'LLVM18.1.2-rust-1.78.0-stable' Reader: 'LLVM 18.1.2-rust-1.78.0-stable')

error: could not compile `core` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
error: could not compile `memchr` (lib) due to 1 previous error
error: could not compile `rustc-demangle` (lib) due to 1 previous error
error: could not compile `alloc` (lib) due to 1 previous error
Build completed unsuccessfully in 0:04:32
Makefile:109: recipe for target '/home/n1/lede/build_dir/target-aarch64_cortex-a53_musl/host/rustc-1.78.0-src/.built' failed
make[3]: *** [/home/n1/lede/build_dir/target-aarch64_cortex-a53_musl/host/rustc-1.78.0-src/.built] Error 1
make[3]: Leaving directory '/home/n1/lede/feeds/packages/lang/rust'
time: package/feeds/packages/rust/host-compile#1682.61#69.14#282.64
    ERROR: package/feeds/packages/rust [host] failed to build.
package/Makefile:114: recipe for target 'package/feeds/packages/rust/host/compile' failed
make[2]: *** [package/feeds/packages/rust/host/compile] Error 1
make[2]: Leaving directory '/home/n1/lede'
package/Makefile:110: recipe for target '/home/n1/lede/staging_dir/target-aarch64_cortex-a53_musl/stamp/.package_compile' failed
make[1]: *** [/home/n1/lede/staging_dir/target-aarch64_cortex-a53_musl/stamp/.package_compile] Error 2
make[1]: Leaving directory '/home/n1/lede'
/home/n1/lede/include/toplevel.mk:229: recipe for target 'world' failed
make: *** [world] Error 2

**Comment by coolsnowwolf** [comment]
内存不足

**Reply by brianzc0306** [reply]
我试试

**Reply by ZqinKing** [reply]
> 内存不足

L大，N1编译能升级以下内核到6.x么？

**Reply by WlphM** [reply]
> 我试试

咋样了，一样的报错
